Container API examples for async upload (#4)
celantur/celantur-examples: README.md

# Celantur: Code Samples 

Here, you find code samples on using Celantur.

## Container API v1 Async Upload/Download

> [!NOTE]
> This functionality is available starting from `24.02.2` version of Container API

> The examples are excessive but split on functions that could be easily copied and accommodated for your needs.

Reviewer: ```suggestion
The examples are split in functions that can be easily copied and accommodated for your needs.
```
